src/CoreWCF.Primitives/src/CoreWCF/Description/ServiceAuthorizationBehavior.cs

```diff
@@ -222,10 +228,7 @@ void IServiceBehavior.ApplyDispatchBehavior(ServiceDescription description, Serv
                             ApplyAuthorizationPoliciesAndManager(behavior);
                         }
 
-                        if (_isAuthorizationServiceSet)
-                        {
-                            behavior.AuthorizationService = _authorizationService;
-                        }
+                        behavior.AuthorizationService = AuthorizationService;
                     }
```

Comment:
Are we going to get build warnings about using obsolete api's? I would prefer to create an internal only api that we use instead of still having code around which uses these public versions. Then we can have the public api's throw when used.

Reply:
done in https://github.com/CoreWCF/CoreWCF/pull/948/commits/3958008c3c678fbc885b94a36361a83d27038b0a